### PR TITLE
OS X compatibility banner (bug 1112275)

### PR DESCRIPTION
**Pull request by mstriemer**
Depends on https://github.com/mozilla/marketplace-core-modules/pull/13.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   ],
   "dependencies": {
     "marketplace-constants": "0.2.1",
-    "marketplace-core-modules": "1.1.0",
+    "marketplace-core-modules": "1.1.1",
```
Comment (ngokevin) [inline bower.json]: i think we should be bumping the minor version and not the patch version

File: src/templates/search/main.html [hunk 2]
```diff
@@ -22,6 +22,7 @@ <h2 class="linefit">
       {% include "compatibility_filtering.html" %}
       <a href="#" class="expand-toggle" title="{{ _('Expand') }}"></a>
     </header>
+    {% include '_includes/os_x_banner.html' %}
```
Comment (ngokevin) [inline src/templates/search/main.html]: nitpick: line breaks around

File: src/templates/_macros/market_tile.html [hunk 3]
```diff
@@ -66,6 +66,9 @@ <h3 itemprop="name">{{ app.name|translate(app) }}</h3>
     {% endif %}
 
     {{ feed_app_tile(feed_app) if feed_app }}
+    {% if is_detail and app.device_types.indexOf('desktop') != -1  %}
```
Comment (ngokevin) [inline src/templates/_macros/market_tile.html]: nitpick: line break above

**Comment by ngokevin** [comment]
i'll apply this once marketplace-core-modules is tagged

**Comment by mstriemer** [comment]
screenshot 2015-01-08 13 41 06
screenshot 2015-01-08 14 35 17
screenshot 2015-01-08 14 38 01

**Comment by ngokevin** [comment]
Could use some padding-top on the app detail page to match the bottom padding

r+wc

**Comment by mstriemer** [comment]
Yeah, the ratings extend beyond their box model unfortunately so it is messing with the padding. I'll see what I can do.

**Comment by ngokevin** [comment]
i think this is okay to close due to UX reworking it

**Pull request closed without merging**